src/StreamerOptions.h

```diff
@@ -1,15 +1,6 @@
-//===-- src/StreamerOptions.h - Streamer consumer options class definition --*-
-// C++ -*-===//
-//
-//
-//===----------------------------------------------------------------------===//
-///
 /// \file This file contains the declaration of the StreamerOptions struct,
-/// which
-/// is used to configure the Streamer and the RdKafka configuration used int the
-/// Streamer. Implementation is in Streamer.cpp
-///
-//===----------------------------------------------------------------------===//
+/// which is used to configure the Streamer and the RdKafka configuration used
+/// int the Streamer. Implementation is in Streamer.cpp
```

Comment:
Rather than saying that the implementation is in Streamer.cpp I think we should refactor it out into it's own cpp file

Reply:
I suppose word `implementation` simply refers to usage. I deleted the comment as it's misleading.